microsoft-edge/webview2/samples/webview2samplewincomp.md

```diff
@@ -6,15 +6,15 @@ ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: webview
-ms.date: 04/27/2022
+ms.date: 06/14/2022
 ---
 # WebView2 sample: Win32 C++ app with Visual Composition
 
 This WebView2 sample demonstrates creating an application that embeds a WebView2 control within a Win32 native application.
 
 It is built as a Win32 Visual Studio 2019 project and makes use of both C++ and HTML/CSS/JavaScript in the WebView2 environment.
 
-It also uses Windows Runtime Composition APIs (also called the Visual layer) to take avantage of the Windows UI features and create a better look, feel, and functionality in C++ Win32 applications.
+It also uses Windows Runtime Composition APIs (also called the Visual layer) to take advantage of the Windows UI features and create a better look, feel, and functionality in C++ Win32 applications.
 
 Directory &amp; .sln: **WebView2SampleWinComp/WebView2SampleWinComp.sln**.
```

Comment:
```suggestion
Directory &amp; `.sln`: **WebView2SampleWinComp/WebView2SampleWinComp.sln**.
```

Reply:
I think file names and extensions should have some sort of formatting ... if this isn't the standard I'm sure you will fix it.